nautobot/core/settings.py

```diff
@@ -188,6 +188,7 @@
     "DEFAULT_RENDERER_CLASSES": (
         "nautobot.core.api.renderers.NautobotJSONRenderer",
         "nautobot.core.api.renderers.FormlessBrowsableAPIRenderer",
+        "nautobot.core.api.renderers.NautobotCSVRenderer",
```

Comment:
nit: Probably alpha sort here

Reply:
This specifies the priority order of renderers, AIUI - I think putting NautobotCSVRenderer at the top would make it so that the default API response (if you don't specify a format or content-type) would become CSV, probably not desirable :-).